app/components/files/file.tsx

```diff
@@ -27,33 +28,43 @@ type FileProps = {
     nonVisibleImagesCount: number;
     onPress: (index: number) => void;
     publicLinkEnabled: boolean;
+    channelName?: string;
+    onOptionsPress?: (index: number) => void;
     theme: Theme;
     wrapperWidth?: number;
+    showDate?: boolean;
     updateFileForGallery: (idx: number, file: FileInfo) => void;
+    asCard?: boolean;
 };
 
 const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
     return {
         fileWrapper: {
             flex: 1,
             flexDirection: 'row',
+            alignItems: 'center',
             marginTop: 10,
             borderWidth: 1,
-            borderColor: changeOpacity(theme.centerChannelColor, 0.4),
-            borderRadius: 5,
+            borderColor: changeOpacity(theme.centerChannelColor, 0.24),
+            borderRadius: 4,
         },
         iconWrapper: {
             marginTop: 7.8,
-            marginRight: 6,
+            marginRight: 7,
             marginBottom: 8.2,
-            marginLeft: 8,
+            marginLeft: 6,
+        },
+        imageVideo: {
+            height: 40,
+            width: 40,
+            margin: 4,
         },
     };
 });
 
 const File = ({
-    canDownloadFiles, file, galleryIdentifier, index, inViewPort = false, isSingleImage = false,
-    nonVisibleImagesCount = 0, onPress, publicLinkEnabled, theme, wrapperWidth = 300, updateFileForGallery,
+    asCard = false, canDownloadFiles, channelName, file, galleryIdentifier, showDate = false, index, inViewPort = false, isSingleImage = false,
```

Comment:
have you checked that this changes have no impact in the post list ?

Reply:
Yes, cards will show the same. Images are still shown.  Files with preview available show the same (Ex. mp4 preview image shows and is playable)

Reply:
Also, a file will still only show the size in the file card.  Probably desinged like this because a file in a post already has the channel (it's in a channel view) and the time is already shown (in the Post View)